DaiQiOS/DaiQiOS/AddTask.swift

```diff
@@ -13,40 +13,82 @@ struct AddTask: View {
     
     var body: some View {
         NavigationView {
-            VStack(alignment: .leading) {
-                TextField("Title", text: $title)
-                Divider()
-                TextEditor(text: $description)
-                    .frame(height: 100)
-                    .cornerRadius(5)
-                    .font(.body)
+            VStack(alignment: .center) {
+                addTitle()
+                Spacer()
+                    .frame(height: 30)
+                addDescription()
                 Spacer()
                 submitAddTaskButton()
+                    .padding()
+                    .toolbar {
+                        ToolbarItem(placement: .principal) {
+                            Text("Add Task")
+                                .font(.title)
+                                .bold()
+                                .foregroundColor(.primary)
+                        }
+                        ToolbarItem(placement: .navigationBarLeading) {
+                            Button("Cancel") {
+                                presentationMode.wrappedValue.dismiss()
+                            }
+                        }
+                    }
             }
-            .padding()
         }
-        .navigationTitle("Add Task")
     }
-    
-    @ViewBuilder
-    private func submitAddTaskButton() -> some View{
-        Button(action: {
-            saveChanges()
-        }) {
-            Text("Submit")
-                .font(.headline)
-                .foregroundColor(.blue)
-                .padding()
-                .background(Color.gray.opacity(0.2))
-                .cornerRadius(30)
-                .shadow(radius: 10)
+    func addTitle() -> some View {
+        VStack(alignment: .leading) {
+            TextField("Title", text: $title)
+                .onChange(of: title) { newValue in
+                    if newValue.isEmpty {
+                        submitButtonDisabled = true
+                    } else {
+                        submitButtonDisabled = false
+                    }
```

Comment:
We can shorten this logic with 
```suggestion
    submitButtonDisabled = newValue.isEmpty
```